src/libs/nav.ts

```diff
@@ -73,13 +73,20 @@ export const getPath = (name: string, params?: Record<string, any>): string => {
 /**
  * alias for getPath()
```

Comment:
Just noticed this docstring. Since `getLink` now includes query parameter functionality that `getPath` does not, can we update this?

Reply:
yes, getPath is the reall workhorse that should get this new logic, and getLink here should just call getPath as a 1-liner.... maybe we can update the type signature to use typeof getPath to enforce this better?

Reply:
Based on the names, one could also argue that `getPath` is only responsible for the pathname part of the URL and `getLink` expands on that. It's unfortunate that Terra UI uses "path" to actually mean "URL" in many places.